src/Stratis.Bitcoin/P2P/Protocol/IMessageListener.cs

```diff
@@ -0,0 +1,13 @@
+﻿/// <summary>
+/// Contract for a recipient part of a consumer logic that handles incoming messages.
+/// </summary>
+/// <typeparam name="T">Type of the messages that are being handled.</typeparam>
+/// <seealso cref="MessageProducer{T}"/>
```

Comment:
You need to add `using Stratis.Bitcoin.P2P.Protocol;` OR use 
```
cref="Stratis.Bitcoin.P2P.Protocol.MessageProducer{T}"
```

so this reference is handled properly. Otherwise resharper creates a "missing reference" warning because it's not resolved.

Reply:
Thank you good find will fix